internal/proxy/force_codex_support.go

```
@@ -29,6 +30,8 @@ const (
 	codexToolSearchProxyName = "tool_search"
 )
 
+var errCodexInputNoConvertibleMessages = errors.New("Codex input contains no convertible messages")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)